.hlint.yaml

```diff
@@ -25,3 +25,27 @@
     Federator.Response,            # this is just a naming conincidence
     Cannon.Run                     # we do something similar, but not identical here by hand
   ] }
+
+# I got these false positives from hlint.  Following the chances leads
+# to (obviously valid) compiler errors:
+#
+# ```
+# libs/wire-api/test/unit/Test/Wire/API/Routes/Version.hs:23:38-111: Warning: Functor law
+# Found:
+#   fmap toLower . show <$> allVersions @=? cs . toByteString'
+#     <$> allVersions
+# Perhaps:
+#   ((fmap toLower . show) . allVersions @=? cs . toByteString'
+#      <$> allVersions)
+#
+# libs/wire-api/test/unit/Test/Wire/API/Routes/Version.hs:31:38-135: Warning: Functor law
+# Found:
+#   tail . show . fromVersionNumber
+#     <$> allVersionNumbers @=? cs . toByteString'
+#     <$> allVersionNumbers
+# Perhaps:
+#   ((tail . show . fromVersionNumber)
+#      . allVersionNumbers @=? cs . toByteString'
+#      <$> allVersionNumbers)
+# ```
+- ignore: { name: Functor law }
```

Comment:
Maybe we don't want to ignore `Functor law` everywhere, but rather where hlint gets tripped up, which seems to be when using the Bilge Assert combinators? 🤔

Reply:
I'd prefer to turn off broken laws.  It's not like either side of the law is wrong, it's merely an aesthetic preference.  Anyone case to weigh in either way?

Reply:
Maybe it's just me who sometimes write "dumb" functor code and waits for hlint to clean it for me. 🤔

Reply:
When I comes to dumb mistakes, I'm a good candidate, too :wink: